app/views/shared/summary_cards/_resource.html.slim

```diff
@@ -55,7 +55,7 @@
       .summary-card__more.summary-card__more--meta
         .summary-card__metadata.summary-card__metadata--last
           i.fa.fa-pencil.glyphicon--right
-          | 102 Annotations
+          = react_component 'TotalAnnotations', {url: resource.url }, class: '__inline'
```

Comment:
Would it make sense to include the `i.fa.fa-pencil` etc in the "Total Annotations" component? Since that icon is part of the component anyway...